code/game/objects/effects/spawners/structure.dm

```diff
@@ -31,3 +31,14 @@ again.
 						/obj/structure/grille,
 						/obj/structure/window/reinforced/fulltile
 						)
+
+/obj/effect/spawner/structure/closet_or_box //75% closet / 25% box, intended for maintenance
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "closet_or_box_spawner"
+
+/obj/effect/spawner/structure/closet_or_box/New()
+	if(prob(25))
```

Comment:
Its :rotating_light: rotating_light use firefox for mobile(android) @duncathan to switch to desktop view easy with one refresh.